sysutils/virtualbox/Makefile

```
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		virtualbox
+PLUGIN_VERSION=		1.0
```

```suggestion
PLUGIN_VERSION=		0.1
PLUGIN_DEVEL=		yes
```

we start with the development version :)

---

Yes, I didn't set the PLUGIN_DEVEL variable because I noticed it is set by default in `../../Mk/plugins.mk`

---

It is, but not for stable branches were we like to „lock down“ development versions to avoid early production testers ;)